### DISCUSSION TITLE
Transfer from ttecks to this Repo

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#13] [Q&A] [answered]
Hi

Is there yet any good way to update existing LXCs from ttecks to the new community-scripts repo?

**Comment by Mellowlynx** [comment]
Not jet, this has to be worked on.
But there is more to figure out and setup, give it some time.

**Reply by CrazyWolf13** [reply]
yes definitely, I tought about looking into creating something, however have not yet started as I did not know if there was already a solution.

**Reply by MickLesk** [reply]
At the moment you can run the bash in LXC instead of “update”. We are working on it.

Example in MySpeed LXC:
`bash -c “$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/myspeed.sh)”`

**Comment by tteck** [comment, marked as answer]
Within the LXC console, execute the following command
```
sed -i 's/tteck\/Proxmox/community-scripts\/ProxmoxVE/g' /usr/bin/update
```

This will switch the repos for the `update` command

**Comment by judeibe** [comment]
@CrazyWolf13 As @tteck  has provided a suitable answer I think you should mark this discussion as answered.

I think a PR is in order to update the Readme and the site.

**Reply by tteck** [reply]
If the LXC is created using this repository, everything is good.
https://github.com/community-scripts/ProxmoxVE/blob/a4a1821822f0f00fde56bc9b1ebe7204f5e01c08/misc/install.func#L201-L202

**Comment by ptrsmk** [comment]
Along with this is the minor issue of updating references to the old website.  For example, upon logging in as root user in my n8n LXC console, a message is displayed: 
n8n LXC provided by https://helper-scripts.com/

Ideally, a solution to updating the update functions would also update such references.

**Reply by tteck** [reply]
Have you gone to  https://helper-scripts.com/?

**Reply by ptrsmk** [reply]
I stand corrected.  I had in mind something said in an earlier thread about that url not changing hands.  Mea culpa.

**Comment by Feramance** [comment]
Is there a similar command for alpine based LXC containers?

**Reply by tteck** [reply]
no

**Comment by DavidMoffatt-web** [comment]
It would be nice to get a script that does this for all LXC's. Perhaps the community can develop one?

**Comment by CrazyWolf13** [comment]
@MickLesk What about the kofi account, will this continue ?

**Reply by MickLesk** [reply]
The Ko-Fi Account is for @tteck. 

I don't yet know whether this will be continued. 
As long as we still can, we can give something back to @tteck. Although it might not help him much, it might help his family <3

**Comment by MickLesk** [comment]
I've made a small script to update the "update" function on all Debian/Ubuntu LXC containers.

Check out the details here: [Pull Request #55](https://github.com/community-scripts/ProxmoxVE/pull/55)

You can run the following command to execute the script:

```bash
bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/misc/update-repo.sh)"
```

![Image of the script output](https://github.com/user-attachments/assets/376846bb-fb31-4685-9a1e-4d9b4f591a7d)
```

**Reply by Mellowlynx** [reply]
Think you mean the **_update-repo_** script, and not the **_update-lxcs_**? 
So: `bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/misc/update-repo.sh)"`
😉

**Reply by MickLesk** [reply]
Right, Copy Paste mistake 😅 updated.

**Comment by Neustradamus** [comment]
@tteck: I have explained the steps to transfer the orginal repo to @community-scripts to have not history/fork/star lost, etc. and there will have redirections:
- 14.3k Stars
- 194 Watchers
- 2.3k Forks

Can you unarchive your original repo and launch the transfer?

I have explained the steps for the @BramSuurdje repository too.

He must to push all changes from current https://github.com/community-scripts/Proxmox to https://github.com/BramSuurdje/proxmox-helper-scripts and remove https://github.com/community-scripts/Proxmox (after pushing), and transfer this repository into the org).

Thanks in advance.

**Reply by judeibe** [reply]
@Neustradamus Is this a transfer of ownership of the repo. If so it was discussed that wasn't happening.

**Reply by Neustradamus** [reply]
@judeibe: Where and why?

Attention, there are two parts:
- @tteck for https://github.com/tteck/Proxmox
- @BramSuurdje for https://github.com/BramSuurdje/proxmox-helper-scripts (upstream of https://github.com/community-scripts/Proxmox)

**Reply by bramsuurdje** [reply]
https://github.com/tteck/Proxmox/discussions/4025#discussioncomment-11121613

**Reply by MickLesk** [reply]
tteck doesn't want it, there's nothing to discuss. That's why we founded the org and detached the fork.

Everything has its reasons.

By the way, tteck is a member too.

**Comment by judeibe** [comment]
Closing this discussion as resolved as the original question was answered.

**Reply by Neustradamus** [reply]
Ok @tteck does not want to transfer.

But the second problem is always here:
- @BramSuurdje for https://github.com/BramSuurdje/proxmox-helper-scripts (upstream of https://github.com/community-scripts/Proxmox)

**Reply by bramsuurdje** [reply]
there are no problems, the site has been moved to this repository and we are just finalizing some things. the repo at my personal github has been archived and the repository at community-scripts/Proxmox will be removed upon the finalization of the remaining tasks.
